.hermes/conveyor/work-3d8d9b32/specs.md

```
@@ -0,0 +1,35 @@
+- Existing `info_maps_to_info` test in `checkstyle.rs` must remain (prevents regression)
+
+## Current State Verification
+
+| Aspect | Status |
+|--------|--------|
+| `Severity::Error` → `"error"` | ✓ Correct |
+| `Severity::Warn` → `"warning"` | ✓ Correct |
+| `Severity::Info` → `"info"` | ✓ Correct |
+| Clippy `match_same_arms` | ✓ No warning |
+| Checkstyle tests | ✓ All 28 pass |
+| Module documentation | ✓ Matches implementation |
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)